src/services/NamespaceService.ts

```diff
@@ -94,8 +94,8 @@ export class NamespaceService {
         const expiredIn = endHeight - namespaceGracePeriodBlocks - currentHeight;
         const deletedIn = endHeight - currentHeight;
         const expiration = expired
-            ? TimeHelpers.durationToRelativeTime(expiredIn, blockGenerationTargetTime)
-            : TimeHelpers.durationToRelativeTime(deletedIn, blockGenerationTargetTime);
```

Comment:
Old logic: If expired then expiration=expired otherwise deleted;
New logic: If expired then expiration=deleted otherwise expired

It doesn't make sense except if you are inverting the logic deliberatelly?

Reply:
Hi @mm-s 
yes, I did it deliberately.

The server return `endHeight` is included namespace grace period blocks (2880 blocks around 1 day).

So the data on `deletedIn` has always 1 day (grace period) extra than `expiredIn `

Example:

![Screenshot 2021-06-11 at 4 16 31 PM](https://user-images.githubusercontent.com/5649156/121654892-c4e0b080-cad0-11eb-914f-cfefc1e15fb8.png)

when creating a new namespace it shows 30 days (it excluded grace period).

So when a namespace is not expired, it should show 30 days rather than 31 days